### ISSUE TITLE
Roadmap

### ISSUE THREAD

**Issue by kfrajtak** [projectstorm/react-diagrams#124]
Hi @dylanvorster,
can you outline the future of this project - what is planned etc.?
Thanks,
Karel

**Comment by sergekol** [comment]
Hi!

Arrows for links (from source to target) and labels for them.

@dylanvorster 
Thank you. I hope that project will grow to a roadmap picture. I believed in it too )))

**Comment by kfrajtak** [comment]
I was thinking about that, I already pushed code (that was merged) for link styling. There's another PR regarding the event system that should be merged before implementing advanced features.

**Comment by dylanvorster** [comment]
• Arrow Links and Arrow link factories
• Keymaps
• Text on links
• Link bridges when links cross
• Group Node examples
• Actual Documentation

**Comment by JokerNN** [comment]
@dylanvorster I managed to draw arrows on links with extending DefaultLinkWidget, I can implement same inside library if you can give me some clues how you see this feature :)

**Comment by matheusgaya** [comment]
any progress on drawing arrow on links?

**Comment by ganesh-sankey** [comment]
Any progress on drawing arrow on links? Is anyone implemented this feature on their fork? Let me know.

**Comment by mstfymrtc** [comment]
@JokerNN is there any progress on the arrow links? Is there any method to achieve this? (built-in or not)? Thanks :)
